### ISSUE TITLE
Does class AvesTorchaudioWrapper(torch.nn.Module): need  self.model.eval()

### ISSUE THREAD

**Issue by ramayer** [ramayer/elephant-rumble-inference#3]
Do I need to set the wrapped model to eval mode too?

Probably.
